### ISSUE TITLE
How to simply output America/New_York?

### ISSUE THREAD

**Issue by 9mm** [marnusw/date-fns-tz#117]
I can't find out to output the actual time zone in a standardized format like `America/New_York`

**Comment by spencerwilson** [comment]
Can you give some example(s) of expected inputs and outputs?

**Comment by 9mm** [comment]
like the standardized time zone: America/New_York

**Comment by 9mm** [comment]
I knew you could do this with Intl but I wasn't sure if thats what date-fns-tz used under the hood and I needed the timezone to match the one detected by date-fns-tz

**Comment by marnusw** [comment]
Unfortunately this is not possible. The Intl API does not support formatting a tz in IANA format since this is typically the input format. i.e. you are probably saving it in this format alongside a date in your database. If you really need to render this you can substitute the input value in the template passed to format as a literal value.

**Comment by 9mm** [comment]
I feel like this is a very common use case (moment-tz.js does it properly for instance). I want to detect the TZ of the browser during API calls to time-series backend (all UTC) of the browser looking at the report. Getting something like "EST/EDT" is not helpful because that's not a usable timezone (for instance if they select 2 year time span where EST/EDT changes multiple times between start and end date).

Currently I'm using this, which works fine: `Intl.DateTimeFormat().resolvedOptions().timeZone`

But I wanted to use `date-fns-tz` to output it instead because I DO use this library to show "PST/PDT" to the user as their detected timezone. If date-fns-tz wasnt using Intl under the hood then these wouldn't match.

Basically I was looking for any output format which is standardized so I can tell the backend the actual TZ of the browser.
